partitura/score.py

```diff
@@ -1477,10 +1476,12 @@ def iter_starting(self, cls, include_subclasses=False):
             Instance of type `cls`
 
         """
-        yield from self.starting_objects[cls]
         if include_subclasses:
-            for subcls in iter_subclasses(cls):
-                yield from self.starting_objects[subcls]
+            for key, items in self.starting_objects.items():
+                if issubclass(key, cls):
+                    yield from items
+        elif cls in self.starting_objects:
+            yield from self.starting_objects[cls]
```

Comment:
I am not familiar with the working of iter_subclasses, but does this change have an impact on the order the objects are yield by iter_all? If so, this would technically be a breaking change, however I don't think that the order is documented anywhere, so I guess it might be OK.

Reply:
Good point. I think the order of objects is likely to change. But as you state, the order of the objects is an undocumented implementation detail, so that shouldn't really keep us from merging this. A warning in the changelog would be prudent though.